pep/app/utils/user.ts

```diff
@@ -36,7 +38,8 @@ export async function onAuthenticated(owner: Controller | Route | Component): Pr
     currentUserService.setFontSize(currentUserService.fontSize.id);
     await themeService.setup();
     await langService.setup();
-    return configurationService.setup();
+    configurationService.setup();
```

Comment:
I think you'll want an `await` here. That being said, I wonder if it might be worth doing a `Promise.all` so these are not occurring synchronously. At the very least for the `themeService`, `langService` and `configurationService` - I don't believe any of those rely on each other.